test/test_document_store.py

```diff
@@ -324,8 +324,9 @@ def test_delete_documents(document_store_with_docs):
     documents = document_store_with_docs.get_all_documents()
     assert len(documents) == 0
 
-
-@pytest.mark.parametrize("document_store_with_docs", ["elasticsearch"], indirect=True)
+    
+@pytest.mark.elasticsearch
+@pytest.mark.parametrize("document_store_with_docs", ["elasticsearch", "memory", "sql", "milvus"], indirect=True)
```

Comment:
If you want to run this test on all document stores, we can simply drop the parametrization line here. Or is there any reason why you exclude FAISS from here?

The "elasticsearch" marker can be dropped in any case. This is from now on only needed if a test is not using a document_store fixture but still requires elasticsearch (see https://github.com/deepset-ai/haystack/blob/master/CONTRIBUTING.md#writing-tests)

Reply:
Not at all, I developed a FAISS test separately in `test_faiss_and_milvus.py` and forgot to update this. Fixing :+1: I haven't seen the new contributing guidelines, there's a lot more in there now! Thanks :)